### PR TITLE
GCC11 Fixes

### PR DESCRIPTION
**Pull request by fabianhjr**
###### Description of changes

enblend-enfuse: 4.2 (2016) → unstable-2022-03-06
vigra: 1.11.1 (2017) → unstable-2022-01-11

No tagged releases in about 5 years, this bump fixes compilation with gcc11 of both packages.

- Built on platform(s)
  - [x] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- [ ] For non-Linux: Is `sandbox = true` set in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
- [ ] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://nixos.org/manual/nixpkgs/unstable/#sec-package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [ ] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [22.05 Release Notes (or backporting 21.11 Release notes)](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md#generating-2205-release-notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
  - [ ] (Release notes changes) Ran `nixos/doc/manual/md-to-db.sh` to update generated release notes
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

### CODE REVIEW

**Review by gador — COMMENTED** [review]
LGTM!

File: pkgs/tools/graphics/enblend-enfuse/default.nix [hunk 1]
```diff
@@ -1,26 +1,39 @@
...
 stdenv.mkDerivation rec {
   pname = "enblend-enfuse";
-  version = "4.2";
+  version = "unstable-2022-03-06";
 
-  src = fetchurl {
-    url = "mirror://sourceforge/enblend/${pname}-${version}.tar.gz";
-    sha256 = "0j5x011ilalb47ssah50ag0a4phgh1b0wdgxdbbp1gcyjcjf60w7";
+  src = fetchzip {
+    url = "https://sourceforge.net/code-snapshots/hg/e/en/enblend/code/enblend-code-0f423c72e51872698fe2985ca3bd453961ffe4e0.zip";
```
Comment (austinbutler) [inline pkgs/tools/graphics/enblend-enfuse/default.nix:24]: This is a 404 for me.

Comment (fabianhjr) [reply pkgs/tools/graphics/enblend-enfuse/default.nix:24]: Weird, commit is still on the tree: https://sourceforge.net/p/enblend/code/ci/0f423c72e51872698fe2985ca3bd453961ffe4e0/tree/

Comment (fabianhjr) [reply pkgs/tools/graphics/enblend-enfuse/default.nix:24]: Clicked on "generate snapshot" or something like that and regenerated the same URL: https://sourceforge.net/code-snapshots/hg/e/en/enblend/code/enblend-code-0f423c72e51872698fe2985ca3bd453961ffe4e0.zip

Comment (fabianhjr) [reply pkgs/tools/graphics/enblend-enfuse/default.nix:24]: ¿Could you retry?

Comment (austinbutler) [reply pkgs/tools/graphics/enblend-enfuse/default.nix:24]: Yeah I went to investigate and ended up at https://sourceforge.net/p/enblend/code/ci/default/tarball, which linked to the same URL.

It seems to work now... hopefully just some temporary stupidity from SF.

**Pull request merged**